frontend/src/components/pages/potions/[slug]/PotionDetails.js

```diff
@@ -0,0 +1,87 @@
+import { Grid } from '@mui/material';
+
+import { getPotionDifficultyColor } from '../../../../lib/utils';
+import AccordionList from '../../../ui/AccordionList';
+import DetailInfoCard from '../../../ui/DetailedInfoCard';
+
+const PotionDetails = ({ attributes, apiLink }) => {
+	const {
+		name,
+		slug,
+		image,
+		effect,
+		difficulty,
+		inventors,
+		side_effects,
+		characteristics,
+		time,
+		ingredients,
+		manufacturers,
+		wiki,
+	} = attributes;
+
+	const accordions = [
+		{
+			name: 'Ingredients',
+			value: ingredients?.split(', '),
+		},
```

Comment:
Maybe we can have 2 or 3 more fields for the accordions, e.g manufacturers, inventors, characteristics etc. Else the accordions is really empty. 
And less things in the informationTable below?